npm-shrinkwrap.json

```diff
@@ -4,5929 +4,5487 @@
   "dependencies": {
```

Comment:
The reason for the 2000 additions and deletions she had was probably because she accidentally updated all our packages.